package.json

```diff
@@ -76,9 +76,11 @@
     "finalhandler": "^1.1.1",
     "get-port": "^4.1.0",
     "jest": "^23.6.0",
+    "jest-extended": "^0.11.0",
     "jsdom": "^13.1.0",
     "klaw-sync": "^6.0.0",
     "nyc": "^13.1.0",
+    "promisify-child-process": "^3.1.0",
```

Comment:
Is this new dependency really necessary? Although the appveyor/windows tests dont run automatically (not sure why) I still check them manually. Cross-spawn has explicit windows support, this dep hasnt. I'd prefer to keep using cross-spawn with the old implementation if possible

Reply:
You're right my bad. I've changed it back to `cross-spawn`, it's actually much nicer!